dungeon/src/contrib/utils/LevelUtils.java

```diff
@@ -35,10 +34,7 @@ public static boolean canSee(
         return false; // if there is a blockFireball in the way, we can't see through
       }
 
-      currentTile =
-          currentTile
-              .level()
-              .tileAt(currentTile.position().add(new Point(direction.x(), direction.y())));
+      currentTile = currentTile.level().tileAt(currentTile.position().translate(direction));
```

Comment:
das ist fies. warum gibt es in Tile nicht eine passende hilfsmethode, die eine direction kriegt? dann kann ich sowas sagen `currentTile = currentTile.tileAt(direction);`.

ist sicher einen extra pr wert.